docs/devel/developer-guides/vagrant.md

```diff
@@ -13,7 +13,7 @@ Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/deve
 
 ### Setup
 
-By default, the Vagrant setup will create a single kubernetes-master and 1 kubernetes-minion. Each VM will take 1 GB, so make sure you have at least 2GB to 4GB of free memory (plus appropriate free disk space). To start your local cluster, open a shell and run:
+By default, the Vagrant setup will create a single kubernetes-master and 1 node (kubernetes-minion). Each VM will take 1 GB, so make sure you have at least 2GB to 4GB of free memory (plus appropriate free disk space). To start your local cluster, open a shell and run:
```

Comment:
Replace first sentence with:
`By default, the Vagrant setup will create a single master node (called kubernetes-master) and 1 node (called kubernetes-minion-1). `